src/math/FFT.hpp

```diff
@@ -0,0 +1,66 @@
+#pragma once
+#include<bits/stdc++.h>
+using namespace std;
+
+//整数配列を複素数へ
```

Comment:
`//` と日本語の間には空白を入れないと文字の色が適用されないようです

![image](https://user-images.githubusercontent.com/28915482/139622965-864c0b02-6f87-48fb-934f-a01b82bd10d2.png)


```suggestion
// 整数配列を複素数へ
```